jiant/tasks/seq2seq.py

```diff
@@ -119,7 +119,8 @@ def get_metrics(self, reset=False):
         return {"perplexity": math.exp(avg_nll), "accuracy": acc}
 
     def update_metrics(self, logits, labels, tagmask=None):
-        self.scorer2(logits.max(2)[1], labels, tagmask)
+        relevant_logits = logits.max(2)[1][:, : labels.shape[1]]
```

Comment:
not sure what is going on with the shapes here, could you leave a comment ? (Also, could you use `logits.max(dim=2)` )

Reply:
Done.